dags/bhr_collection.py

```diff
@@ -128,41 +128,4 @@
         },
         dag=dag,
     )
-
-    bhr_collection_gcs_test = SubDagOperator(
-        task_id="bhr_collection_gcs_test",
-        dag=dag,
-        subdag=moz_dataproc_pyspark_runner(
-            parent_dag_name=dag.dag_id,
-            image_version="1.5-debian10",
-            dag_name="bhr_collection_gcs_test",
-            default_args=default_args,
-            cluster_name="bhr-test-collection-{{ ds }}",
-            job_name="bhr-test-collection",
-            python_driver_code="https://raw.githubusercontent.com/mozilla/python_mozetl/main/mozetl/bhr_collection/bhr_collection.py",
-            init_actions_uris=[
-                "gs://dataproc-initialization-actions/python/pip-install.sh"
-            ],
-            additional_metadata={
-                "PIP_PACKAGES": "boto3==1.16.20 click==7.1.2 google-cloud-storage==2.7.0"
-            },
-            additional_properties={
-                "spark:spark.jars": "gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar",
-            },
-            py_args=[
-                "--date",
-                "{{ ds }}",
-                "--sample-size",
-                "0.5",
-                "--use_gcs",
-            ],
-            idle_delete_ttl=14400,
-            num_workers=6,
-            worker_machine_type="n1-highmem-4",
-            gcp_conn_id=params.conn_id,
-            service_account=params.client_email,
-            storage_bucket=params.storage_bucket,
-        ),
-    )
```

Comment:
If you intend to try this again next week, you might consider simply commenting this out instead.